dev.yml

```diff
@@ -5,6 +5,9 @@ up:
   - ruby: 2.3.1
   - bundler
 
+commands:
+  test:
```

Comment:
What do you think about including the ability to specify a test file like we do in [nginx-routing-modules](https://github.com/Shopify/nginx-routing-modules/blob/1962ec1fc78b2621ea0f122031819a9d54ec9882/dev.yml#L38-L48)?